### DISCUSSION TITLE
Measuring DevOps Success- in practice

### DISCUSSION THREAD

**Discussion by tracymiranda** [cdfoundation/end-user-council#2] [Show and tell]
How does your team/organization measure DevOps success? Is it measuring DORA metrics? If so, how? What other metrics are measured and how is this done?

Please share your real world examples here 

Example:
  * I work at XYZ corp on the ABC product (roughly 15 person team)
  * We measure DORA metrics through a homegrown solution managed by a centralized team. 
  * We also measure employee satisfaction with a quarterly NPS survey administered through <name of tool>

**Comment by jsalinas29** [comment]
https://www.linkedin.com/posts/opsera_2021-devops-predictions-by-garima-bajpai-activity-6757725176747692032-GYAR/
